tests/test_data.py

```diff
@@ -1311,63 +1311,88 @@ def test_load_chmap(self):
-            [ax.annotate(str(recording_sites[ipt]), (aligned_np_drive_coordinates[ipt,0], aligned_np_drive_coordinates[ipt,1]), ha='center', va='center', color='r') for ipt in range(len(recording_sites))]
+            [ax.annotate(str(acq_elecs[ipt]), (drive2_coordinates[ipt,0], drive2_coordinates[ipt,1]), ha='center', va='center', color='k',fontsize=5) for ipt in range(len(acq_elecs))]
+            [ax.annotate(str(recording_sites[ipt]), (aligned_np_drive_coordinates[ipt,0], aligned_np_drive_coordinates[ipt,1]), ha='center', va='center', color='r', fontsize=5) for ipt in range(len(recording_sites))]
+            ax.set(xlim=(-8,8), ylim=(-8,8), title=f'{subject}')
+            visualization.base.overlay_sulci_on_spatial_map(subject, 'LM1', 'ECoG244', theta=0, offset=-drive2_offset)
+            ax.set_aspect('equal')
             visualization.savefig(write_dir, f'{neuropixel_drive}_{drive2}_alignment_{subject}.png')
 
             neuropixel_drive='NP_Insert137'
-            aligned_np_drive_coordinates, recording_sites = align_recoring_drives(neuropixel_drive, drive2, subject)
+            aligned_np_drive_coordinates, drive2_coordinates , recording_sites, acq_elecs = align_recoring_drives(neuropixel_drive, drive2, subject, drive2_offset=drive2_offset)
             fig, ax = plt.subplots(1,1)
-            visualization.plot_ECoG244_data_map(np.zeros(256,), cmap='Greys', ax=ax)
-            visualization.base.annotate_spatial_map_channels(drive_type=drive2, color='k', ax=ax)
-            [ax.annotate(str(recording_sites[ipt]), (aligned_np_drive_coordinates[ipt,0], aligned_np_drive_coordinates[ipt,1]), ha='center', va='center', color='r') for ipt in range(len(recording_sites))]
-            visualization.base.overlay_sulci_on_spatial_map(subject, 'LM1', 'ECoG244', theta=0)
-            ax.set(xlim=(-2,13.25), ylim=(-2,13.25))
+            [ax.annotate(str(acq_elecs[ipt]), (drive2_coordinates[ipt,0], drive2_coordinates[ipt,1]), ha='center', va='center', color='k',fontsize=5) for ipt in range(len(acq_elecs))]
+            [ax.annotate(str(recording_sites[ipt]), (aligned_np_drive_coordinates[ipt,0], aligned_np_drive_coordinates[ipt,1]), ha='center', va='center', color='r', fontsize=5) for ipt in range(len(recording_sites))]
+            visualization.base.overlay_sulci_on_spatial_map(subject, 'LM1', 'ECoG244', theta=0, offset=-drive2_offset)
+            ax.set_aspect('equal')
+            ax.set(xlim=(-8,8), ylim=(-8,8), title=f'{subject}')
             visualization.savefig(write_dir, f'{neuropixel_drive}_{drive2}_alignment_{subject}.png')
+        
+        # Test that neuropixel offset works
+        subject='affi'
+        neuropixel_drive = 'NP_Insert72'
+        drive2 = 'ECoG244'
+        aligned_np_drive_coordinates, drive2_coordinates ,  recording_sites, acq_elecs = align_recoring_drives(neuropixel_drive, drive2, subject, neuropixel_drive_offset=-drive2_offset)
```

Comment:
add theta=90 for affi example

Reply:
why? The plot it produces is correct

Reply:
to test whether rotating the whole map after alignment works